tasks/kmt.py

```diff
@@ -720,9 +951,8 @@ def build(
     if arch is None:
         arch = "local"
 
-    if vms is None:
-        vms = ",".join(stacks.get_all_vms_in_stack(stack))
-        info(f"[+] Running tests on all VMs in stack {stack}: vms={vms}")
+    if not os.path.exists(layout):
+        raise Exit(f"File {layout} does not exist")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Could be replaced by asserts